.github/ISSUE_TEMPLATE/default.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Default Issue Template
+about: Capture context and user stories
+labels: 
+---
+
+# User Story
+- <!-- Why does this need to be done? Who will it benefit and how? -->
+
+# Description/Context
```

Comment:
Add a section for designs/ux.

Reply:
I'm still a bit unclear on what you mean by a design section... Is the intent to capture links to e.g. Invision projects, screenshots,... ?